### PR TITLE
Enhancement: NU Improvements

### PR DESCRIPTION
**Pull request by Batorian**

### CODE REVIEW

File: plugins/english/novelupdates.ts [hunk 1]
```diff
@@ -379,7 +395,7 @@ class NovelUpdates implements Plugin.PluginBase {
           .first()
           .html()!;
         if (!chapterContent) {
-          chapterContent = `${loadedCheerio('#chapter-body').html()!}<hr><br>There could be missing content, please check in webview.`;
+          chapterContent = `${loadedCheerio('#chapter-body').html()!}<hr><nr>There could be missing content, please check in webview.`;
```
Comment (rajarsheechatterjee) [inline plugins/english/novelupdates.ts:398]: `<nr>`?

Comment (Batorian) [reply plugins/english/novelupdates.ts:398]: Thx, that's not meant to be there.

**Comment by Batorian** [comment]
- added bloat removal for `scribblehub`

**Review by rajarsheechatterjee — APPROVED** [review]

**Comment by Batorian** [comment]
- added `Femme Fables` as supported site

**Comment by Batorian** [comment]
- added `Redox Translation` as supported site

**Comment by Batorian** [comment]
- fixed search

File: src/plugins/english/novelupdates.ts [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
-import { fetchApi, fetchFile } from '@libs/fetch';
+import { fetchApi } from '@libs/fetch';
 import { Filters, FilterTypes } from '@libs/filterInputs';
 import { Plugin } from '@typings/plugin';
 
 class NovelUpdates implements Plugin.PluginBase {
   id = 'novelupdates';
   name = 'Novel Updates';
-  version = '0.5.4';
+  version = '0.6.2';
```
Comment (nyagami) [inline src/plugins/english/novelupdates.ts:9]: it should be 0.60 or 0.5.5

Comment (Batorian) [reply src/plugins/english/novelupdates.ts]: Updated it.

**Review by nyagami — APPROVED** [review]

**Pull request merged**